docs/standard/components.md

```diff
@@ -1,80 +1,88 @@
+The.NET Framework is the original .NET implementation that has existed since 2002. It's the same .NET Framework that existing .NET developers have always used. Versions 4.5 and later implement the .NET Standard, so code that targets the .NET Standard can run on those versions of the .NET Framework. It contains additional Windows-specific APIs, such as APIs for Windows desktop development with Windows Forms and WPF. The .NET Framework is optimized for building Windows desktop applications.
 
 To learn more about the .NET Framework, see the [.NET Framework Guide](../framework/index.md).
 
-### Mono for Xamarin
+### Mono
 
-Mono is the runtime used by Xamarin apps.  It implements the .NET Standard, which means that any code that targets the .NET Standard can run on Xamarin apps.  It contains additional APIs for iOS, Android, Xamarin.Forms, and Xamarin.Mac.  It is optimized for building mobile applications on iOS and Android.
+Mono is the .NET implementation used by Xamarin apps. It implements the .NET Standard, so code that targets the .NET Standard can run on Xamarin apps. It contains additional APIs for iOS, Android, Xamarin.Forms, and Xamarin.Mac. It's optimized for building mobile applications on iOS and Android.
 
 To learn more about Mono, see the [Mono documentation](http://www.mono-project.com/docs/).
 
-## .NET tooling and common infrastructure
+### Universal Windows Platform (UWP)
+
+UWP is an implementation of .NET that is used for building modern, touch-enabled Windows applications and software for the Internet of Things (IoT). It's designed to unify the different types of devices that you may want to target, including PCs, tablets, phablets, phones, and even the Xbox. UWP provides many services, such as a centralized app store, an execution environment (AppContainer), and a set of Windows APIs to use instead of Win32 (WinRT). Apps can be written in C++, C#, VB.NET, and JavaScript. When using C# and VB.NET, the .NET APIs are provided by .NET Core.
 
-Tooling for .NET is also common across each implementation of .NET.  These include, but are not limited to:
+To learn more about UWP, see [Intro to the Universal Windows Platform](https://docs.microsoft.com/windows/uwp/get-started/universal-application-platform-guide).
+
+## .NET runtimes
+
+A runtime is the execution environment for a managed program. The OS is part of the runtime environment but is not part of the .NET runtime. Here are some examples of .NET runtimes:
+ 
+ - Common Language Runtime (CLR) for the .NET Framework
+ - Core Common Language Runtime (CoreCLR) for .NET Core
+ - .NET Native for Universal Windows Platform 
```

Comment:
- The Mono runtime for Xamarin.iOS, Xamarin.Android, Xamarin.Mac, and the Mono desktop framework.